app/component/project_components/project_card_component.html.erb

```diff
@@ -0,0 +1,53 @@
+<div class="row center-row search-card-container">
+  <div class="col-12 col-sm-12 col-md-5 col-lg-5">
+    <div class="row center-row">
+      <%= image_tag project_image_preview(project, current_user), alt: project.name, class: "search-image" %>
```

Comment:
@VaibhavUpreti We no longer need to pass current_user as we completely rolled out the S3 feature, right ?

Reply:
yes, we need to remove the feature flag @senbo1 as the migration is completed.